types/react-router/test/examples-from-react-router-website/README.md

```diff
@@ -2,4 +2,4 @@ Examples taken from react-router website.
 
 Please keep them in sync and as close as possible to the original ones.
 
-See https://github.com/ReactTraining/react-router/tree/22bd52c901ef0312348c2b02549c7102bd5653ae/packages/react-router-website/modules/examples
+See https://github.com/ReactTraining/react-router/tree/master/packages/react-router-dom/examples
```

Comment:
Could you use a permalink? Otherwise this might get unavailable.

Reply:
How do you mean? Are you suggesting the `master` branch would go away?

Reply:
No but the path. Using a commit SHA (like before) or a tag would guarantee that the content is available.

Reply:
Right, but wouldn't it also mean the info could be out of date?

I think `master` is better than any `SHA` personally.

Reply:
Better old than unavailable. With a sha we know whether a type was changed or a mistake was made originally.

Reply:
SHAs can still be force pushed off. I’d argue master remaining the default branch is more permanent even than any SHA in that repositories history.

Nothing is permanent. If nothing is permanent, we might as well be up-to-date.

Reply:
> SHAs can still be force pushed off. 

Nobody is doing that on master.

Reply:
If you think that master is better then try replacing the sha in the previous link with `master` and tell me how this improved the link.

Reply:
Well, I changed the link cuz it was broken for me, but I have an idea I think you’ll love.

Reply:
So, I'm not sure what should be done, but leaving it behind seems a little irresponsible 🤔

Reply:
Okay, I'm hoping this will suffice! Thanks for the reviews.